tests/foreman/ui/test_activationkey.py

```diff
@@ -967,17 +960,13 @@ def test_positive_add_multiple_aks_to_system(session, module_org, rhel6_contenth
                 {'name': key_name, 'lce': {env_name: True}, 'content_view': cv_name}
             )
             assert session.activationkey.search(key_name)[0]['Name'] == key_name
-            session.activationkey.add_subscription(key_name, product_name)
-            ak = session.activationkey.read(key_name, widget_names='subscriptions')
-            subscriptions = [
-                subscription['Repository Name']
-                for subscription in ak['subscriptions']['resources']['assigned']
-            ]
-            assert product_name in subscriptions
+            ak = session.activationkey.read(key_name, widget_names='repository sets')[
+                'repository sets'
+            ]['table'][0]
+            assert product_name == ak['Product Name']
         # Create VM
         rhel6_contenthost.install_katello_ca(target_sat)
         rhel6_contenthost.register_contenthost(module_org.label, ','.join([key_1_name, key_2_name]))
```

Comment:
I think we could flip this to the global registration within scope of this PR (it is used on 8 other places in this module).
Also, IIRC RHEL6 support will be dropped this summer and I believe that PRT will fail to checkout RHEL6. We should rather proceed with `rhel_contenthost` and parametrize the test for RHEL8 or 9. And f-string.

Reply:
There is an ongoing effort to replace these with global registration: https://github.com/SatelliteQE/robottelo/pull/14034
But please make the changes if possible and I can rebase my PR.